src/note.rs

```diff
@@ -261,6 +261,12 @@ impl Note {
         &self.value_commitment
     }
 
+    /// Returns the encryped data from the note as an array of
+    /// BlsScalar's
+    pub fn cipher(&self) -> [BlsScalar; CIPHER_SIZE] {
+        *self.encrypted_data.cipher()
```

Comment:
This should match the signature of the actual `cipher` method that we're proxying, so it should be: 
```rust
pub fn cipher(&self) -> &[BlsScalar; CIPHER_SIZE] {
    self.encrypted_data.cipher()
}
```